packages/amplify-category-auth/src/provider-utils/awscloudformation/question-factories/core-questions.ts

```diff
@@ -49,12 +49,9 @@ export const parseInputs = async (
   const prefix = input.prefix ? `${'\n'} ${questionChalk(input.prefix)} ${'\n'}` : '';
 
   let question: $TSAny = {
```

Comment:
don't have full context on where this is used, but it looks like it's still formatted to be an input to inquirer?

Reply:
wrote this helper method to handle the return object from parseInputs: https://github.com/lazpavel/amplify-cli/blob/auth-inquirer-to-prompter/packages/amplify-category-auth/src/provider-utils/awscloudformation/service-walkthroughs/auth-questions.ts#L718-L757